core/state/state_object.go

```diff
@@ -112,6 +114,7 @@ func newObject(db *StateDB, address common.Address, acct *types.StateAccount) *s
 		originStorage:  make(Storage),
 		pendingStorage: make(Storage),
 		dirtyStorage:   make(Storage),
+		isFilled:       make(map[common.Hash]bool),
```

Comment:
This is a nit irrelevant for now but could be `map[common.Hash]struct{}` to save space. There's no need to distinguish "empty entry" form "false" value.